containers/daap-python-base/src/var/task/data_product_metadata.py

```diff
@@ -332,23 +418,71 @@ class to handle creation and updating of
     schema json files relating to data products as a whole
     """
 
-    # returns path of latest metadata or v1 if it doesn't exist
+    MINOR_UPDATE_FIELDS = {
```

Comment:
Changing any data product level (non schema level) fields which are editable by the user will trigger a minor version update I think. Looking through the spec, only schema changes would be a breaking change for the user.